src/routes/event.js

```diff
@@ -0,0 +1,13 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
```

Comment:
Benefits of splitting your code into small functions:
1) easier to spot bugs
2) easier to test
3) easier for other developers to read
4) easier for future maintenance/improvements  
...

I suggest that you do that